suite/tests/common/decode.c

```diff
@@ -94,7 +94,7 @@ my_setjmp(sigjmp_buf env)
 }
 
 static void
-signal_handler(int sig)
+signal_handler(int sig, siginfo_t *siginfo)
```

Comment:
A signal handler should either have 1 arg (signal()) or 3 (sigaction()): I don't think there is any such thing as one taking 2 args.

Reply:
Fixed.